### DISCUSSION TITLE
Excited for 2.4! Estimated Release Date?

### DISCUSSION THREAD

**Discussion by CloudyJudgment** [Security-Onion-Solutions/securityonion#10031] [2.4]
I'm excited for the migration to 2.4 and the integration of the Elastic Agent. Is there an estimated release date at this time?

**Comment by dougburks** [comment]
Security Onion 2.4 Beta 1 Release is now available!
https://blog.securityonion.net/2023/03/security-onion-24-beta-release-now.html

**Reply by ColeVan** [reply]
Is there a link for the 2.4 ISO?

**Reply by TOoSmOotH** [reply]
There is no ISO for beta 1. You need to install a rocky linux MINIMAL install. The ISO is not quite ready yet due to how elastic agent is packaged making the ISO 11GB so we are working on getting that down.

**Reply by ColeVan** [reply]
No worries. I've spun up a rocky linux minimal server and now installing SO 2.4. Hopefully all goes well. Thanks for the response!! You all rock!
